src/app/locales/vi_VN.ts

```diff
@@ -0,0 +1,513 @@
+import {LocalizedLabels, LocalizedSteps, registerLocalization} from '../localization';
+
+const uiLabels: LocalizedLabels = {
+  'Angular Update Guide': 'Hướng dẫn Update Angular',
+  'Select the options matching your project:': 'Tuỳ chỉnh những lựa chọn cho phù hợp với dự án của bạn:',
+  'Angular Versions': 'Phiên bản Angular',
+  From: 'Từ',
+  To: 'Đến',
+  Warning: 'Chú ý',
+  'Plans for releases after the current major release are not finalized and may change. These recommendations are based on scheduled deprecations.':
+    'Đối với những phiên bản Angular được phát hành sau phiên bản chính thức hiện tại, kế hoạch và dự định cho những phiên bản này chưa được thông qua chính thức và có thể sẽ thay đổi. Những gợi ý sau dựa trên những chỉnh sửa đã được thống nhất.',
+  'We do not currently support downgrading versions of Angular.':
+    'Chúng tôi hiện tại không hỗ trợ Downgrade phiên bản Angular',
+  'We do not recommend moving across multiple major versions.':
+    'Chúng tôi không khuyến khích update Angular theo kiểu "nhảy cóc" phiên bản.',
+  'App Complexity': 'Độ phức tạp của ứng dụng',
+  Basic: 'Cơ bản',
+  Medium: 'Vừa',
+  Advanced: 'Nâng cao',
+  'Show update information relevant to all Angular developers.':
+    'Hiển thị thông tin update dành cho tất cả mọi tầng lớp lập trình viên Angular.',
```

Comment:
Might be worth replacing all instances of ``update`` with ``cập nhật``

Reply:
I'd agree with this. I've worked on this PR over the span of some hours in the morning and some hours in the afternoon which might result in some differences between translations. Thanks a lot for taking your time to review this though.

Reply:
Changed to "'Hiển thị thông tin update dành cho tất cả lập trình viên Angular ở mọi cấp độ.'"